android-components/components/feature/search/src/main/java/mozilla/components/feature/search/storage/BundledSearchEnginesStorage.kt

```diff
@@ -37,9 +37,10 @@ internal class BundledSearchEnginesStorage(
     override suspend fun load(
         region: RegionState,
         locale: Locale,
+        distribution: String?,
         coroutineContext: CoroutineContext,
     ): SearchMiddleware.BundleStorage.Bundle = withContext(coroutineContext) {
-        val localizedConfiguration = loadAndFilterConfiguration(context, region, locale)
+        val localizedConfiguration = loadAndFilterConfiguration(context, region, locale, distribution)
         val searchEngineIdentifiers = localizedConfiguration.visibleDefaultEngines
```

Comment:
`val searchEngineIdentifiers = localizedConfiguration.visibleDefaultEngines`

Everything looks good apart from this naming convention now. Since it might not be `visibleDefaultEngines` everytime, we should probably think of a more generic one.. thoughts?

Reply:
That's a good point, I'll try out some names to see if that makes the code more clear.  Thanks

Reply:
I've renamed it to `visibleSearchEngines`  Thanks